src/model/entities/media.ts

```diff
@@ -1,4 +1,5 @@
 export interface Media {
   id: string
+  mediaId: string
```

Comment:
How does `mediaId` differ from `id`?
Isn't `mediaId` "just" the id of the Media i.e. the Media.id?

With this naming you are going to have code looking like `const mediaId: string = media.mediaId` as opposed to `const mediaId: string = media.id`. That's a lot of Media ;)